crossing-repair.lic

```diff
@@ -10,18 +10,72 @@ class CrossingRepair
   include DRCT
 
   def initialize
```

Comment:
Thanks for reformatting with `initialize` and arg parsing at the top. Right or wrong, the consistency with the other scripts makes it easier (for me at least) to reason about the script and know where to look for things